README.md

```diff
@@ -208,75 +208,75 @@ import hudson.plugins.sshslaves.verifiers.NonVerifyingKeyVerificationStrategy
 import com.cloudbees.plugins.credentials.*
 import com.cloudbees.plugins.credentials.domains.Domain
 import hudson.model.*
-import com.amazon.jenkins.ec2fleet.EC2FleetCloud
+import com.amazon.jenkins.ec2fleet.FleetCloud
 import jenkins.model.Jenkins
 
 // just modify this config other code just logic
 config = [
-    region: "us-east-1",
-    // EC2 Spot Fleet ID
-    // or Auto Scaling Group Name
-    fleetId: "...", 
-    idleMinutes: 10,
-    minSize: 0,
-    maxSize: 10,
-    numExecutors: 1,
-    awsKeyId: "...",
-    secretKey: "...",
-    ec2PrivateKey: '''-----BEGIN RSA PRIVATE KEY-----
+        region       : "us-east-1",
```

Comment:
Looks like the new indentations were added accidentally?

Reply:
Yeah not sure why that happened, but they are reverted

Reply:
Don't think this change was reverted. Unresolving conv to make sure the change gets done.